app/models/game_session.rb

```diff
@@ -0,0 +1,38 @@
+class GameSession < ApplicationRecord
+  TOTAL_LIVES = 9
+  WORDS = %w[programming fun internship ruby rails coding design].freeze
+
+  before_create :set_word
+
+  def set_word
+    self.word_to_guess = WORDS.sample
+  end
```

Comment:
I think we can move this to private, since it's not something we want any outside class to have access to at the moment.